Dockerfile_generator

```diff
@@ -7,16 +7,6 @@ RUN go install github.com/vektra/mockery/v2@v2.13.0 && \
     git clone https://github.com/kubernetes/code-generator -b v0.26.1 --depth 1 && \
```

Comment:
Do we need the Dockerfile_generator any more if we do not have the code generation?

Reply:
I actually don't know why we need that file for. If you think there is no use, I can remove it too.

Reply:
ah right, it was for generating code inside the container like in NFD. yes, then we should remove it too.